PatientWebApplication/HealthClinic.DAL/Model/ActionsAndBenefits/Message.cs

```diff
@@ -10,32 +10,37 @@ public class Message : Entity
 
         public DateTime TimeStamp { get; set; }
 
+        public string DateStamp { get; set; }
+
         public bool IsRemoved { get; set; }
 
         public string PharmacyName { get; set; }
 
         public string DateAction { get; set; }
 
+
         public Message() : base()
         {
         }
 
-        public Message(int id, string text, DateTime timestamp, bool isremoved, string pharmacyname, string dateaction) : base(id)
+        public Message(int id, string text, string datestamp, DateTime timestamp, bool isremoved, string pharmacyname, string dateaction) : base(id)
         {
             Text = text;
             TimeStamp = timestamp;
             IsRemoved = isremoved;
             PharmacyName = pharmacyname;
             DateAction = dateaction;
+            DateStamp = datestamp;
         }
 
-        public Message(string text, DateTime timestamp, bool isremoved, string pharmacyname, string dateaction)
+        public Message(string text, string datestamp, DateTime timestamp, bool isremoved, string pharmacyname, string dateaction)

```

Comment:
Chnage names of variables to match lines above, for example datestamp should be dateStamp, pharmacyname should be pharmacyName...